src/reaction_network/vertex.hpp

```diff
@@ -186,7 +191,7 @@ Vertex::Vertex(
 
   std::string formula = SBML_formulaToString(reaction.getKineticLaw()->getMath());
```

Comment:
We should check if the return of ```reaction.getKineticLaw()->getMath()``` is ok before passing it to SBML_formulaToString